nemo/collections/asr/parts/numba/__init__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
How about naming the folder rnnt_numba under parts and drop the numba top folder?

Reply:
Addressed below